cmd/imagec/imagec.go

```diff
@@ -296,7 +304,115 @@ func ImagesToDownload(manifest *Manifest, storeName string) ([]*ImageWithMeta, e
 		}
 	}
 
-	return images, nil
+	return images, imageLayer, nil
+}
+
+// Will updated metadata based on the current image being requested
+//
+// Currently on tags are updated, but once imagec supports pulling by digest
+// then digest will need to be updated as well
+func updateImageMetadata(imageLayer *ImageWithMeta, manifest *Manifest) error {
+	// if standalone no need to continue
+	if options.standalone {
+		return nil
+	}
+	// updated Image slice
+	var updatedImages []*ImageWithMeta
+
+	// assuming calls to the portlayer are cheap, lets get all existing images
+	existingImages, err := ListImages(imageLayer.Store, []*ImageWithMeta{})
```

Comment:
was asking whether calling this function like following would be possible

```
existingImages, err := ListImages(imageLayer.Store, nil)
```

instead of 

```
existingImages, err := ListImages(imageLayer.Store, []*ImageWithMeta{})
```